src/client/actions/GuildScheduledEventUpdate.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const Action = require('./Action');
+const { Events } = require('../../util/Constants');
+
+class GuildScheduledEventUpdateAction extends Action {
+  handle(data) {
+    const client = this.client;
+    const guild = client.guilds.cache.get(data.guild_id);
+
+    if (guild) {
+      const oldGuildScheduledEvent = guild.scheduledEvents.cache.get(data.id)._clone() ?? null;
```

Comment:
```suggestion
      const oldGuildScheduledEvent = guild.scheduledEvents.cache.get(data.id)?._clone() ?? null;
```
If the event isn't cached it will cause an error on this line.

Reply:
can't find that end part in the documentation

Reply:
> can't find that end part in the documentation

@iShibi It seems like it's only sent for the `GUILD_CREATE` event
![from startup](https://i.imgur.com/IyTfOc1.png)

Reply:
If there isn't already a PR for this can you open one at https://github.com/discord/discord-api-docs

Reply:
Update: https://github.com/discord/discord-api-docs/pull/4139 got merged, so the field can be added now.